react-native-plain-text/src/PlainText.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { requireNativeComponent, TextInput, StyleSheet } from 'react-native';
+
+const emptyFunctionThatReturnsTrue = () => true;
+
+class PlainText extends TextInput {
```

Comment:
Sure! That sounds like a good idea.

Should we rename everything then? Even the library `react-native-wp-text-input`? or similar

Reply:
Subclassing a component isn't a problem per say, but in this case it's a bit problematic since we don't maintain `TextInput`. It's internal implementation can easily change which would break our code each time we update react-native. Could you tell me a bit more about the requirements for `PlainText`, what does it have that `TextInput` doesn't?